### PR TITLE
Remove unnecessary session creation

### PR DESCRIPTION
**Pull request by cristianoperez**
Solves the problem described in https://github.com/caelum/vraptor4/issues/1088

## Changelog
Remove unnecessary session creation, also this could cause problems if google bots starts to hit you very hard.

### CODE REVIEW

File: vraptor-core/src/main/java/br/com/caelum/vraptor/core/JstlLocalization.java [hunk 1]
```diff
@@ -119,7 +120,7 @@ private Object findByKey(String key) {
 			return value;
 		}
 
-		value = Config.get(request.getSession(), key);
+		value = Config.get(request.getSession(false), key);
```
Comment (Turini) [inline vraptor-core/src/main/java/br/com/caelum/vraptor/core/JstlLocalization.java]: @cristianoperez thanks! it sounds better as a default value.

I'd only keep a fallback strategy for users that for some reason want a new session to be created. 
what do you think?

for example:
```
request.getSession(createNewSession())

//...
protected boolean createNewSession(){
    return false;
}
```

what do you think?

File: vraptor-core/src/main/java/br/com/caelum/vraptor/core/JstlLocalization.java [hunk 2]
```diff
@@ -51,9 +51,10 @@
 
 	private final HttpServletRequest request;
 
-	/** 
+	/**
 	 * @deprecated CDI eyes only
 	 */
+	@Deprecated
```
Comment (Turini) [inline vraptor-core/src/main/java/br/com/caelum/vraptor/core/JstlLocalization.java]: no need for this one. we use only the javadoc comment by convention (;

Comment (cristianoperez) [reply vraptor-core/src/main/java/br/com/caelum/vraptor/core/JstlLocalization.java]: eclipse things

File: vraptor-core/src/main/java/br/com/caelum/vraptor/core/JstlLocalization.java (hunk 1 above)
Comment (cristianoperez) [reply vraptor-core/src/main/java/br/com/caelum/vraptor/core/JstlLocalization.java]: done

**Pull request merged**